lib/auth/auth_with_roles.go

```diff
@@ -4137,25 +4137,6 @@ func checkInventorySupportsRole(ctx context.Context, role types.Role, authVersio
 }
 
 func minRequiredVersionForRole(role types.Role) (semver.Version, string, error) {
-	// DELETE IN 15.0.0
-	// The label expression checks can be deleted in 15.0.0 since all servers
-	// that don't support label expressions are on 13.x or older. If no other
-	// feature checks have been added at that time, checkInventorySupportsRole
```

Comment:
should we just delete `checkInventorySupportsRole` entirely? It's really not great, the inventory checks are very flaky

Reply:
I had that in mind but I though I shouldn't.
Removed in [646e0b3](https://github.com/gravitational/teleport/pull/36419/commits/646e0b3101ab3bb9c9c95787d5694f6f64444859)